style/ruby/README.md

```diff
@@ -40,6 +40,63 @@ text editor implementations for Rubocop and Reek.
    directory. We check ours into `.gitignore` since we use the one here via
    [Code Climate](https://codeclimate.com).
 
+**Command Line with Pronto**
+
+With Pronto, you can locally run the same suite as CodeClimate from the comfort of your terminal.
+
+1. Install `pronto`.
```

Comment:
Can you update this markdown to add indentation to the code blocks so it supports the ordered list. Right now, we just have a few step 1s 😃 

The following markdown should accomplish what we want!

```
1. Install `pronto`.

    ```bash
    gem install pronto
    ```

1. Take care of any dependencies it might say you lack.

1. Install the following gems.

    ```bash
    gem install pronto-rubocop pronto-reek
    ```

1. Create a config file in your local directory so you can tell Pronto what to ignore.

    ```bash
    touch .pronto.yml
    ```
```

Reply:
@licatajustin hmmm... I think I refactored it just as you suggested, but I'm not sure if it was successful.

Reply:
Looks right to me! 💯